pdr_backend/lake/etl.py

```diff
@@ -218,10 +234,16 @@ def _calc_bronze_start_end_ts(self):
             self.raw_table_names, TableType.TEMP
         ).values()
 
+        fin_timestamp = (
+            self.ppss.lake_ss.fin_timestr
+            if self.ppss.lake_ss.fin_timestr != "now"
+            else datetime.now().strftime("%Y-%m-%d_%H:%M")
```

Comment:
i don't think this is right, we need to query the raw_tables to figure out how far the ETL can run

Reply:
OKay I got the issue now. I have changed the `_get_max_timestamp_values_from` method and it does not query the table if it does not exists. So I am going to remove it